### DISCUSSION TITLE
👀 .Net Developer

### DISCUSSION THREAD

**Discussion by gondar** [AzureCommunityPL/group#191] [Oferty pracy]
💸 20 000 - 24 000 PLN gross senior B2B 💸
💸14 000 - 18 000 PLN gross mid-level B2B 💸
🌍 Remote

Who's up for a challenge?

Requirements (it's about skills not technologies for us):
➡ Familiarity with Azure PaaS offering
➡ You know how to write clean code (you know SOLID, TDD and best practices)
➡ You’re unwaveringly customer-focused
➡ You’re agile – willing to wear different hats depending on needs
➡ You’ll have some success stories to share

Read more:
😎 https://guestline.bamboohr.com/jobs/view.php?id=89&source=aWQ9Mw%3D%3D

**Comment by kamil-mrzyglod** [comment]
Te widełki to z VAT czy netto? Bo dla mnie _gross_ oznaczało zawsze kwotę brutto, która zawiera w sobie VAT ;)

**Reply by gondar** [reply]
B2B dla firmy zagranicznej wiec VAT wynosi 0%, czyli i jedno i drugie. Masz racje, gross to brutto, ale w tym przypadku bez znaczenia.
